src/meta/processors/parts/AlterSpaceProcessor.cpp

```diff
@@ -0,0 +1,90 @@
+/* Copyright (c) 2018 vesoft inc. All rights reserved.
+ *
+ * This source code is licensed under Apache 2.0 License.
+ */
+
+#include "meta/processors/parts/AlterSpaceProcessor.h"
+
+namespace nebula {
+namespace meta {
+void AlterSpaceProcessor::process(const cpp2::AlterSpaceReq& req) {
+  folly::SharedMutex::WriteHolder wHolder(LockUtils::spaceLock());
+  const std::vector<std::string>& zones = req.get_paras();
+  const std::string& spaceName = req.get_space_name();
+  cpp2::AlterSpaceOp op = req.get_op();
+  switch (op) {
+    case cpp2::AlterSpaceOp::ADD_ZONE: {
+      nebula::cpp2::ErrorCode ret = addZones(spaceName, zones);
+      if (ret != nebula::cpp2::ErrorCode::SUCCEEDED) {
+        handleErrorCode(ret);
+        onFinished();
+        return;
+      }
+      break;
+    }
+    default:
+      break;
+  }
+  handleErrorCode(nebula::cpp2::ErrorCode::SUCCEEDED);
+  onFinished();
+}
+
+nebula::cpp2::ErrorCode AlterSpaceProcessor::addZones(const std::string& spaceName,
```

Comment:
Could add some comments and blank lines between code segments?

Reply:
> Could add some comments and blank lines between code segments? It's hard to read all lines with on explict segments in one breath.

apparently these code is clear and easy, not every code need comments,  I think

Reply:
It's easy or not do not depend what you think, but the people who read it. Because you have written it not long, you surely think it easy.

Reply:
done